templates/taxi/driver_detail.html

```diff
@@ -7,7 +7,12 @@ <h1>
 
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
-  <p><strong>License number:</strong> {{ driver.license_number }}</p>
+  <p>
+    <strong>License number:</strong> {{ driver.license_number }} 
+    <a href="{% url 'taxi:driver-license-update' pk=driver.id %}" class="btn btn-secondary link-to-page">
+      Update License
+    </a>
+  </p>
   <p><strong>Is staff:</strong> {{ driver.is_staff }}</p>
```

Comment:
Consider whether displaying the `is_staff` status is necessary in the driver detail view. If this information is not relevant to the users of this page, it might be better to omit it.